src/console_cmds.cpp

```diff
@@ -1312,6 +1313,21 @@ DEF_CONSOLE_CMD(ConGetDate)
 	return true;
 }
 
+DEF_CONSOLE_CMD(ConGetSysDate)
+{
+	if (argc == 0) {
+		IConsoleHelp("Returns the current date (day-month-year) of your system. Usage: 'getsysdate'");
+		return true;
+	}
+
+	time_t t;
+	struct tm * timeinfo;
+	time(&t);
+	timeinfo = localtime(&t);
+	IConsolePrintF(CC_DEFAULT, "System Date: %d/%d/%d %d:%d:%d", timeinfo->tm_year + 1900, timeinfo->tm_mon, timeinfo->tm_mday, timeinfo->tm_hour, timeinfo->tm_min, timeinfo->tm_sec);
```

Comment:
I think I'd prefer yyyy-mm-dd, to make it more closely resemble getdate (and also ISO8601)